guides/common/assembly_configuring-an-alternate-cname.adoc

```diff
@@ -0,0 +1,6 @@
+[id='configuring-an-alternate-cname_{context}']
+= Configure an Alternate CNAME for {Project}
+You can configure an alternate CNAME for {Project}. This can be useful if <placeholder>.
```

Comment:
@evgeni can you suggest what this feature can be useful for in real life?

Reply:
This is useful if you want to run the {Project} web interface on a different name than the one used by client systems to connect to {Project}.